libpurecool/dyson.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, email, password, country):
         self._country = country
         self._logged = False
         self._auth = None
-        self._headers = {'User-Agent': DYSON_API_USER_AGENT}
+        self._headers = {'User-Agent': DYSON_API_USER_AGENT, 'Content-Type': 'application/json'}
```

Comment:
```suggestion
        self._headers = {'User-Agent': DYSON_API_USER_AGENT}
```
When using `json` param instead of `data`, this header will be added by `requests` automatically so not needed.

Reply:
Resolved in 6307f4c